src/Common/PlatformLinux.inl

```diff
@@ -1097,6 +1097,15 @@ inline void restore_path_separators(char * path)
     while (char* sep = strchr(path, '/')) *sep = '\\'; //
 }
 
+/** Replace windows path separator on Linux path separator 
+ * @brief normalize_path_separators
+ * @param path
+ */
+inline void normalize_path_separators(char * path)
+{
+    while (char* sep = strchr(path, '\\')) *sep = '/'; //
+}
```

Comment:
What? This is a full duplicate of `convert_path_separators` function!

Reply:
I didn't see this function :(
Replaced